portfolio/src/main/webapp/script.js

```diff
@@ -15,7 +15,7 @@
 /** @var slideIndex slide currently being displayed */
 var slideIndex = 1;
 
-/** @const ADD_ATTRIBUTE used to add a space in class attributes */
+/** @const ADD_ATTRIBUTE !string used to add a space in class attributes */
```

Comment:
nit: The comment describes what it the code is doing not what its for/why it is needed. Does that make sense? I think you can write: used for adding an attribute. Or something of that nature. Not need to write adding a space.

Reply:
Makes sense, fixed.